test/integration/servicecluster.go

```diff
@@ -123,7 +123,7 @@ func newServiceClusterSuite(
 			},
 		}
 		require.NoError(t, managementClient.Create(ctx, catalogEntrySet))
-		require.NoError(t, testutil.WaitUntilReady(ctx, managementClient, catalogEntrySet, testutil.WithTimeout(2*time.Minute)))
+		require.NoError(t, testutil.WaitUntilReady(ctx, managementClient, catalogEntrySet, testutil.WithTimeout(time.Minute)))
```

Comment:
Why do you change this timeout back?

Reply:
because I thought that the problem was with `catalogentryset` cleanup, but apparently it also needs to be increased here